Occasional crashes, SIGSEGV

Owncast v0.1.1 (running on Ubuntu Server 22.04) crashes occasionally while leaving this in the log:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x591d00]
goroutine 40 [running]:
net/url.(*URL).String(0x0)
        /usr/local/go/src/net/url/url.go:800 +0x40
github.com/owncast/owncast/activitypub/inbox.handleLikeRequest({0x1ae05e0?, 0x0?}, {0x4ece0e8, 0xc00096a840})
        /build/activitypub/inbox/like.go:16 +0x95
github.com/go-fed/activity/streams.JSONResolver.Resolve.func1({0xc00107e32c, 0x4})
        /go/pkg/mod/github.com/owncast/activity@v1.0.1-0.20211229051252-7821289d4026/streams/gen_json_resolver.go:559 +0x3885
github.com/go-fed/activity/streams.JSONResolver.Resolve({{0xc000e85320?, 0x1b1?, 0xc0006a9e00?}}, {0x4eaf288, 0xc00013e038}, 0xc00031d710)
        /go/pkg/mod/github.com/owncast/activity@v1.0.1-0.20211229051252-7821289d4026/streams/gen_json_resolver.go:888 +0x171
github.com/owncast/owncast/activitypub/resolvers.Resolve({0x4eaf288, 0xc00013e038}, {0xc00095b200, 0x1b1, 0x200}, {0xc000e85320?, 0x0?, 0xc00000e678?})
        /build/activitypub/resolvers/resolve.go:35 +0x159
github.com/owncast/owncast/activitypub/inbox.handle({0xc000e01b00, {0xc000af65e6, 0x6}, {0xc00095b200, 0x1b1, 0x200}})
        /build/activitypub/inbox/worker.go:32 +0x21d
github.com/owncast/owncast/activitypub/inbox.worker(0xc0000341e0?, 0xc000114540?)
        /build/activitypub/inbox/workerpool.go:40 +0x108
created by github.com/owncast/owncast/activitypub/inbox.InitInboxWorkerPool
        /build/activitypub/inbox/workerpool.go:26 +0x51
owncast.service: Main process exited, code=exited, status=2/INVALIDARGUMENT
```
I had this with v0.0.13 already for some weeks now. Since this version was outdated already, I just looked at the issues here and saw that a similar issue was reported (and closed) already. So I didn't give it much attention. But after updating to v0.1.1 this keeps going on.
When I turn off the Fediverse features, everything is fine and no crash happens. But since a lot of my viewers use Fediverse auth, it's not the best option to turn that off all the time. :)
I hope this report helps. If you need further info, please ask.

--- a/activitypub/inbox/like.go
+++ b/activitypub/inbox/like.go
@@ -13,6 +13,14 @@ import (
 func handleLikeRequest(c context.Context, activity vocab.ActivityStreamsLike) error {
 	object := activity.GetActivityStreamsObject()
 	actorReference := activity.GetActivityStreamsActor()
+	if object.Len() < 1 {
+		return errors.New("like activity is missing object")
+	}
+
+	if actorReference.Len() < 1 {
+		return errors.New("like activity is missing actor")
+	}
+
 	objectIRI := object.At(0).GetIRI().String()
 	actorIRI := actorReference.At(0).GetIRI().String()
 

